src/unix/fsevents.c

```diff
@@ -291,13 +289,31 @@ static void uv__fsevents_event_cb(ConstFSEventStreamRef streamRef,
 
       memset(event, 0, sizeof(*event));
       memcpy(event->path, path, len + 1);
-
+      
+#ifndef MAC_OS_X_VERSION_10_7
       if ((eventFlags[i] & kFSEventsModified) != 0 &&
           (eventFlags[i] & kFSEventsRenamed) == 0)
         event->events = UV_CHANGE;
       else
         event->events = UV_RENAME;
-
+#else
+      if (handle->type_flag == kUVFSEventsDirectory) {
+        if ((eventFlags[i] & kFSEventsModified) != 0 &&
```

Comment:
sure, this was the previous behavior before this change. Do you want me to add a comment in the code?